rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.additional.spec.ts

```diff
@@ -0,0 +1,271 @@
+import {
+  rundeckServiceMock,
+  rundeckClientMock,
+  axiosMock,
+  i18nMocks,
+  mockEventBus,
+} from "./mock";
+import { shallowMount, VueWrapper } from "@vue/test-utils";
+import ActivityList from "../activityList.vue";
+import ActivityFilter from "../activityFilter.vue";
+import OffsetPagination from "../../../../library/components/utils/OffsetPagination.vue";
+import { setupRundeckContext } from "./setupRundeckContext";
+import { GlobalOptions } from "./type";
+jest.mock("../../../../library/rundeckService", () => rundeckServiceMock);
+jest.mock("@rundeck/client", () => rundeckClientMock);
+jest.mock("axios", () => axiosMock);
+type ActivityListInstance = InstanceType<typeof ActivityList>;
+const shallowMountActivityList = async (
+  props = {},
+  globalOptions: GlobalOptions = {}
+) => {
+  const wrapper = shallowMount(ActivityList, {
+    props: {
+      eventBus: mockEventBus,
+      displayMode: "full",
+      ...props,
+    },
+    global: {
+      components: {
+        OffsetPagination,
+        ActivityFilter,
+      },
+      stubs: {
+        modal: true,
+        ActivityFilter: true,
+        Modal: {
+          template: "<div class='modal-stub'><slot></slot></div>",
+        },
+        ProgressBar: { template: "<div><slot></slot></div>" },
+        btn: {
+          template: `<button :disabled="disabled" data-test-id="activity-list-delete-selected-executions">Delete Selected Executions</button>`,
+          props: ["disabled"],
+        },
+
+        tooltip: true,
+        ...globalOptions.stubs,
+      },
+      directives: {
+        tooltip: () => {},
+      },
+      mocks: {
+        ...i18nMocks,
+        ...globalOptions.mocks,
+      },
+      ...globalOptions,
+    },
+  });
+  await wrapper.vm.$nextTick();
+  return wrapper as VueWrapper<ActivityListInstance>;
+};
+beforeAll(() => {
+  jest.useFakeTimers();
+  setupRundeckContext();
+});
+afterAll(() => {
+  jest.useRealTimers();
+});
+afterEach(() => {
+  jest.clearAllMocks();
+});
+describe("ActivityList Component", () => {
+  it("renders the component correctly", async () => {
+    const wrapper = await shallowMountActivityList();
+    expect(wrapper.exists()).toBe(true);
+  });
```

Comment:
Please avoid adding tests like this in the future. This has been a repeated ask in the PRs.

Reply:
> Please avoid adding tests like this in the future. This has been a repeated ask in the PRs.

I apologize for the oversight. The test in question was initially included to ensure that all necessary imports and mocks were correctly set up. However, I understand that this is not a recommended practice and it has been a repeated point of feedback in the PRs. I appreciate your patience and I will ensure to avoid including such tests in the future. Thank you for bringing this to my attention.